release-please-config.json

```diff
@@ -10,6 +10,7 @@
             "extra-files": [
                 "README.md",
                 "Provider/build.gradle.kts"
+                "Provider/src/main/java/com/spotify/confidence.ConfidenceFeatureProvider.kts"
```

Comment:
Would this even work? And how bad practice is this?
It's nice to imagine a solution based off of release-please that is easy and works the same way in all providers, but this method would have a few important back-draws